docs/manual/common/guide/devmode/index.toc

```diff
@@ -1,6 +1,6 @@
 DevEnvironment:Development Environment
 RunningServices:Running Services
-ServicePort:How are ports assigned to services?
+ConfiguringServicesInDevelopment:How Lagom services configured in development?
```

Comment:
Missing word "are" (should be "How are Lagom services configured in development?")